### PR TITLE
S2 Illustrated message width token update

### PR DESCRIPTION
**Pull request by PaliwalSparsh**
## Description
The following changes were made to the component layout tokens for both mobile and desktop.

**New tokens**
illustrated-message-vertical-maximum-width
illustrated-message-horizontal-maximum-width

**Deprecated token**
illustrated-message-maximum-width (remapped to illustrated-message-vertical-maximum-width)

## Motivation and context
Added tokens needed for the new S2 illustrated message.

## Related issue
https://jira.corp.adobe.com/browse/SDS-13730

## Types of changes

<li> [ ] Patch (bug fixes, typos, mistakes; non-breaking change which fixes an issue) </li>
<li> [X] Minor (add a new token, changing a value, deprecating a token; non-breaking change which adds functionality) </li>
<li> [ ] Major (deleting a token, changing token value type, renaming a token by deprecating the old one; fix or feature that would cause existing functionality to change) </li>

## Checklist:

<li> [X] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html). </li>
<li> [X] I updated the token in all applicable sets. This applies if updating, adding, or deleting a token that has data across different sets (for example, if the value differs across color themes.) </li>

### CODE REVIEW

**Comment by mrcjhicks** [comment]
**[TESTING spectrum2](https://github.com/adobe/spectrum-tokens-studio-data/actions/runs/10405738784)**

READING spectrum-tokens: `/spectrum-tokens/packages/tokens/src/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum2-colors/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum2-non-colors/`

  > WARNING - STUDIO MISSING MODE DEFINITION FOR spectrum2/component/Component/standard-panel/gripper-drag: ["light"]
  > WARNING - STUDIO MISSING MODE DEFINITION FOR spectrum2/component/Component/standard-panel/gripper-color-drag: ["dark"]
  > WARNING - STUDIO INCLUDES DEPRECATED TOKEN: overlay-opacity/dark [31d5b502-6266-4309-8f8a-3892e6e158da]
  > WARNING - STUDIO INCLUDES DEPRECATED TOKEN: overlay-opacity/light [26b9803c-577f-4a29-badd-dfc459e8b73e]


<details><summary>STUDIO SYNC SUMMARY</summary><p>

```
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component.illustrated-message-horizontal-maximum-width
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component.illustrated-message-vertical-maximum-width
```
</p></details>

<details><summary>SPECTRUM SYNC SUMMARY</summary><p>

```
NEW TOKENS: (2) illustrated-message-horizontal-maximum-width, illustrated-message-vertical-maximum-width
DEPRECATED TOKENS: (1) illustrated-message-maximum-width
CHANGED VALUES: (1) illustrated-message-maximum-width impacting 2 tokens
CHANGED SCHEMAS: (1) illustrated-message-maximum-width
```
</p></details>

TEST COMPLETE

File: src/tokens-studio/spectrum2-non-colors/spectrum2/layout.component/desktop.json [hunk 1]
```diff
@@ -2659,8 +2659,8 @@
       }
     }
   },
-  "illustrated-message-maximum-width": {
-    "value": "380px",
+  "🚫 illustrated-message-maximum-width": {
+    "value": "{illustrated-message-vertical-maximum-width}",
```
Comment (karstens) [inline src/tokens-studio/spectrum2-non-colors/spectrum2/layout.component/desktop.json:2663]: @PaliwalSparsh @lynnhao @GarthDB @mrcjhicks 
This is an example of a working deprecation. the new token `illustrated-message-vertical-maximum-width` has the same value `380px`. This is okay to do. 👍

**Review by karstens — APPROVED** [review]
LGTM. Thank you!

**Review by larz0 — APPROVED** [review]
Reviewed and approved 👍

**Pull request merged**